### DISCUSSION TITLE
student pack request help

### DISCUSSION THREAD

**Discussion by 1PokerMan** [community/community#30074] [GitHub Education]
someone help me, I try to request the student developer pack, but it rejects me with a legend:
* You are more likely to be verified if you have completed your profile

I already modified and completed my profile but it's still the same, upload several photos of my university notebook, exam certificates and the poster persists, thanks

**Comment by manasesjesus** [comment]
Qué tal Pablo! 

Go to Settings > Emails and provide your university email. Personal or work emails aren't allowed; it must be from an educational institution, in your case the utn.edu.ar. Make sure it's added and verified in your emails: https://github.com/settings/emails

You need to activate location services on your browser and go to this webpage: https://education.github.com/discount_requests/student_application
Providing your current location helps GitHub to verify your affiliation with your chosen school/university. You are far less likely to be verified if you choose not to share your location.

Provide the other requested data, and that's it. You'll get the GitHub student benefits for two years. After that, if you're still a student, you can renew it.

Buena suerte!

**Reply by manasesjesus** [reply]
PS: I had it four years while being a student.
<img width="321" alt="gsp" src="https://user-images.githubusercontent.com/24204142/185619291-261315c6-5b4d-446f-88e9-94042cc3a05f.png">

**Reply by 1PokerMan** [reply]
Thanks, I tried what you tell me but that legend persists

**Reply by mariajorrat** [reply]
> Thanks, I tried what you tell me but that legend persists
I have the same problem.
